oshdb-api/src/main/java/org/heigit/ohsome/oshdb/api/mapreducer/MapReducer.java

```diff
@@ -2143,18 +2150,13 @@ private void checkTimeout() {
     // basic optimizations (“low hanging fruit”):
     // single filters, and-combination of single filters, etc.
     if (filter instanceof TagFilterEquals) {
-      OSMTag tag = this.getTagTranslator().getOSMTagOf(((TagFilterEquals) filter).getTag());
-      return mapRed.osmTag(tag);
-    }
-    if (filter instanceof TagFilterEqualsAny) {
-      OSMTagKey key = this.getTagTranslator().getOSMTagKeyOf(
-          ((TagFilterEqualsAny) filter).getTag());
+      return mapRed.osmTag(((TagFilterEquals) filter).getTag());
+    } else if (filter instanceof TagFilterEqualsAny) {
+      OSHDBTagKey key = ((TagFilterEqualsAny) filter).getTag();
       return mapRed.osmTag(key);
-    }
-    if (filter instanceof TypeFilter) {
+    } else if (filter instanceof TypeFilter) {
       return mapRed.osmType(((TypeFilter) filter).getType());
```

Comment:
The usage of `osmType` seems to be deprecated here.

Reply:
right. But the deprecation is very likely not going to remove the `osmType` method, but only make it `private`. I didn't find a good way to represent this kind of deprecation with the `@Deprecated` annotation… :thinking:  Maybe we should add a private, non-deprecated `osmTypeInternal` and use that one both here and from the to-be-removed `osmType` method?

Reply:
We don't have to do that now in this PR.